.devcontainer/README.rst

```diff
@@ -1,15 +1,16 @@
+.. _develop_on_containers:
 
-
-======================
-Develop on a container
-======================
+=======================
+Develop on a containers
+=======================
```

Comment:
```suggestion
Develop on containers
=====================
```
If you want plural "containers", then you want to omit the "a" article.

Reply:
Upss.. that was an obvious one :(